python/paddle/fluid/tests/unittests/mkldnn/test_matmul_v2_bf16_mkldnn_op.py

```diff
@@ -0,0 +1,257 @@
+#   Copyright (c) 2021 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import print_function
+
+import unittest
+import numpy as np
+
+from paddle.fluid.tests.unittests.op_test import OpTest, convert_float_to_uint16
+import paddle.fluid.core as core
+import paddle
+import paddle.fluid as fluid
+import paddle.fluid.framework as framework
+
+
+def reference_matmul(X, Y, transpose_X=False, transpose_Y=False):
```

Comment:
Is there possibility that both test classes common parts can be unified in more general way ?

Reply:
Done, I have unified them. Further unification is not possible due to timeout limit in CI